.travis.yml

```diff
@@ -0,0 +1,17 @@
+os:
+ -osx
+ -linux
+ 
+language: r
+r:
+ - "3.1"
+
+language: python
```

Comment:
I don't think these `language:` blocks can be stacked up on the same level. I was struggling with that when I added support for testing R and Python together in [uptasticsearch's travis.yml](https://github.com/UptakeOpenSource/uptasticsearch/blob/master/.travis.yml#L25).

Could you try something like what I did there with `matrix: include:`?

Also, it would be TOTALLY fine if for this PR you only did python and we ad `language:r` stuff in a later PR